jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -284,6 +293,26 @@ private void checkNonSunProxyMemberAccess(SecurityManager security, ClassLoader
 	}
 }
 
+/*
+ * This helper method atomically writes the given \p fieldvalue to the 
+ * field specified by the \p fieldname
+ */
+private long getFieldOffsetAndCacheFieldValue(String fieldname, Object fieldvalue){
+	long fieldOffset;
+	try {
+		Field field = Class.class.getDeclaredField(fieldName); //$NON-NLS-1$
```

Comment:
`//$NON-NLS-1$` is not needed here. This field doesn't match the incoming parameter, pls verify the code generated by JPP still compiles.